pkgs/development/python-modules/patool/default.nix

```diff
@@ -36,7 +36,7 @@ buildPythonPackage rec {
 
   prePatch = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
Why is prePatch better than postPatch ? Changed it anyway.

Reply:
We want to change the file after patches has been applied. Every package that uses prePatch like this potentially breaks patches.